### ISSUE TITLE
GlobalTag: question

### ISSUE THREAD

**Issue by mcepeda** [uwcms/FinalStateAnalysis#488]
Hi everybody,

Are the GlobalTags used centralized currently? (They used to be in the environment.sh, but I do not see them now).

If they arent, I think we should add them and monitor them centrally. 

(In https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_76X/NtupleTools/test/make_ntuples_cfg.py#L241 , we pick them by default right?)

Right now it does not matter too much, but it will next week when the new JEC corrections arrive. 

Best,

Maria

**Comment by dntaylor** [comment]
Yes, autocond lets the file tell you what to use. You can always manually change it with the --globalTag option. We took out the environment.sh globaltags back when 50ns and 25ns required different globalTags.

**Comment by mcepeda** [comment]
I see that the --globalTag can be set by the user, but I think it would be nice to do this so that it is centralized using the JETMET latest recomendations. for 76X 25ns at least. I see chaos in the future otherwise :). 
But OK, we can handle it on the HTauMu side internally. 
Maria
